OrbitSsh/DirectTcpChannelManager.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitSsh/DirectTcpChannelManager.h"
+
+#include <memory>
+#include <string>
+
+#include "OrbitBase/Logging.h"
+#include "OrbitSsh/Channel.h"
+#include "OrbitSsh/ResultType.h"
+
+namespace OrbitSsh {
+
+DirectTcpChannelManager::DirectTcpChannelManager(Session* session_ptr,
+                                                 int remote_port,
+                                                 std::string remote_host)
+    : channel_(session_ptr),
+      remote_port_(remote_port),
+      remote_host_(remote_host) {}
+
+DirectTcpChannelManager::State DirectTcpChannelManager::Tick() {
```

Comment:
Tick is not very descriptive.. this is a state machine - ProcessState?

Reply:
I added documentation. Maybe different naming makes sense. Let me know